src/apps/bot/handlers.py

```diff
@@ -119,21 +117,10 @@ async def no_fund(update: Update, context: ContextTypes.DEFAULT_TYPE):
 
 
 async def new_fund_form(update: Update, context: ContextTypes.DEFAULT_TYPE):
-    await update.callback_query.answer()
-    await update.callback_query.delete_message()
-    await context.bot.send_message(
-        chat_id=update.effective_chat.id,
-        text="Нажмите на кнопку ниже, чтобы заполнить анкету",
-        reply_markup=ReplyKeyboardMarkup.from_button(
-            KeyboardButton(
-                "Заполнить анкету",
-                # TODO: заменить на веб-приложение с формой
-                # Данные для подстановки в форму:
-                # context.user_data[AGE] - возраст
-                web_app=WebAppInfo(url="https://python-telegram-bot.org/static/webappbot"),
-            )
-        ),
-    )
+    webapp_url_new_fund = (
+        f"https://msppbot.duckdns.org"
```

Comment:
Тут используй `WEBHOOK_URL` (и ниже тоже) 
```suggestion
        f"https://{setings.WEBHOOK_URL}"
```

Reply:
Насчет загадочного вебхука , который в  моем webapp будет смотреться как седло на корове, лучше сделать понятную всем переменную окружения типа DOMAIN или (если с протоколом сразу надо) то APPLICATION_URL и использовать ее и в webapp и в webhook ( это даже более логично), про самодокументирование кода что-то слышали? Это когда сущностям дают максимально понятные названия, чтобы сразу было понятно, что эта сущность делает )))  Что делает вебхук из названия не понятно, даже после курса обучения 14 месяцев по программе Python-разработчик plus, меня этому не учили, к сожалению.